README.md

```diff
@@ -171,6 +171,7 @@ local rx = radix.new({
 |>       |greater than|{"arg_age", ">", 24}|
 |<       |less than  |{"arg_age", "<", 24}|
 |~~      |Regular match|{"arg_name", "~~", "[a-z]+"}|
+|IN      |find in array |{"arg_name", "IN", {"1","2"}}|
```

Comment:
I prefer `in`.

Reply:
i have the same doubt,
I choose `IN` rather than `in`
becos I think the operator code here is like a SQL,  `IN`  is a keyword , usually we use capital style with keyword

Reply:
python is `in`:
https://www.w3schools.com/python/ref_keyword_in.asp

js is `in` too: https://www.cnblogs.com/GreenLeaves/p/6397824.html

Reply:
both `in` and `IN` is allowed for now